develop/integration/rdk-b-22q3/meta-rdk/recipes-containers/librbusprovider/librbusprovider.bb

```diff
@@ -0,0 +1,21 @@
+SRC_URI = "git://git@gitlab.consult.red/dave.chapman/rdk-smart-router.git;protocol=ssh;branch=main \
```

Comment:
Is this a publicly accessible site?
How is it licensed?  (LICENSE set to append "" is unusual.)
(If not publicly accesible, then we have a recipe in an opensourced repo referring to somewhere private.)

Reply:
This should not have been included in the pull request. We now have all recipes as pending merges into existing meta-layers. The recipes were removed by a previous pull request which I merged. We're pushing some improvements from our local copy of DSM to the hosted on here. Unfortunately I wasn't clear in stating it should only be the code under the src directory that should be pushed. Will either close this pull request, or fix so its only code updates, and there are no more recipes here ... as it should be.